widgets/nuxeo-file.html

```diff
@@ -141,23 +217,59 @@
         }
 
         _updateValue() {
-          this.value = {
-            'upload-batch': this.batchId,
-            'upload-fileId': (this.files.length - 1).toString(),
-          };
+          if (this.multiple) {
+            if (!this.value || !Array.isArray(this.value)) {
+              this.value = [];
+            }
+            for (let i = 0; i < this.files.length; i++) {
+              this.push('value', {
+                'upload-batch': this.batchId,
+                'upload-fileId': i.toString(),
+                _name: this.files[i].name,
+                _data: URL.createObjectURL(this.files[i]),
```

Comment:
I don't feel very comfortable in having these private properties (`_name` and `_data`) on the value, since they're going to be sent to the server without need. What do you thing about removing these from value and instead computed these values on the `_fileName` and `_data` helpers?